src/core/HTTPTransport.ts

```diff
@@ -0,0 +1,82 @@
+export enum Method {
+  GET = "GET",
+  PUT = "PUT",
+  POST = "POST",
+  DELETE = "DELETE",
+}
+
+interface Options {
+  method: Method;
+  data?: any;
+  timeout?: number;
+  headers?: Record<string, string>;
+}
+
+function queryStringify(data: string) {
+  return Object.entries(data)
+    .map(([key, value]) => `${key}=${value}&`)
+    .join('')
```

Comment:
```suggestion
    .map(([key, value]) => `${key}=${value}`)
    .join('&')
```
так будет немного чище